back/.prettierrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "tabWidth": 4
```

Comment:
yeah we gonna need loads more rules :D

Reply:
quotestyle 
indent rules 
newline rules

member ordering 
import ordering
my recommendations

Reply:
Do you have a sample .prettierrc.json file I could reuse? Or do you want to open a PR with it?

Reply:
sry i mostly use eslint or tslint and the one project i use prettier is using the js configuration